src/status_im/ui/screens/signing/views.cljs

```diff
@@ -239,14 +243,26 @@
               {:type  :small
                :title :t/send-request-amount
                :error amount-error
-               :accessories [[acc-text (if amount (str amount) "0")
-                              (or display-symbol fee-display-symbol)]]}]
-             [separator]])
-          [list-item/list-item
-           {:type        :small
-            :title       :t/network-fee
-            :error       gas-error
-            :accessories [[acc-text fee fee-display-symbol] :chevron]
+               :accessories [react/nested-text {:style {:color colors/gray}} 
+                            [{:style {:color colors/black}} (if amount (str amount) "0")]
+                             " "
+                             (or display-symbol fee-display-symbol)
+                             " · " 
+                             [{:style {:color colors/black}} (if converted-value (str "~" (i18n/format-currency converted-value (:code wallet-currency)))
+                                                                   [react/activity-indicator {:color   :colors/gray
+                                                                                              :ios   {:size  :small}
+                                                                                              :android {:size :16}}])]
+                             " "
+                             (str (:code wallet-currency))]}
+             [separator]]
+              [list-item/list-item
```

Comment:
careful with parens you slurped one too many here, list-item is not part of the accessories

Reply:
Thanks for catching that.  I meant to change this back to WIP as I'm struggling to get the style props to cooperate for the vectors in my nested-text element where they override the global style though maybe that misplaced parens is the issue.  I'll check back in on it later tonight and see if that resolves it.